src/contexts/auth-context.js

```diff
@@ -0,0 +1,19 @@
+import {createContext, useContext, useState,} from 'react'
+
+const AuthContext = createContext()
+
+const AuthContextProvider = ({children}) => {
+    const [authCred, setAuthCred] = useState({
+        authToken : localStorage.getItem("TOKEN"),
+        authStatus : localStorage.getItem("TOKEN") ? true : false,
```

Comment:
here you can use NOT operator twice, like this
```js
authStatus : !!localStorage.getItem("TOKEN")
```
this converts a value to boolean type